web/src/test/java/com/epam/rft/atsy/cucumber/settings/positions/PositionsStepDefs.java

```diff
@@ -22,118 +24,228 @@
 import cucumber.api.java.en.When;
 
 public class PositionsStepDefs {
+  private DriverProvider driverProvider;
   private String positionsListContents;
   private String positionName;
 
-  @And("^the user is on an Atsy page$")
-  public void the_user_is_on_an_Atsy_page() throws Throwable {
-    assertThat(getDriver().getCurrentUrl(), containsString("secure"));
+  public PositionsStepDefs(DriverProvider driverProvider) {
+        this.driverProvider = driverProvider;
   }
 
-  @When("^the Beállítások menu point clicked$")
-  public void the_button_clicked() throws Throwable {
-    getDriver().findElement(By.id("settings_link")).click();
+  // Anita: ez nem szerepelt a feature fájlban úh innen is kivettem
```

Comment:
Please remove the code in comments.

If it is really necessary, we can look them up later from the git history.